pumpkin/src/server/mod.rs

```diff
@@ -113,6 +116,15 @@ impl Server {
                 self.server_listing.lock().await.add_player();
             }
         }
+
+        // Handle join message
+        let msg_txt = format!("{} joined the game.", player.gameprofile.name.as_str());
+        let msg_comp = TextComponent::text(msg_txt.as_str()).color_named(NamedColor::Yellow);
+        for player in world.current_players.lock().await.values() {
+            player.send_system_message(&msg_comp).await;
+        }
+        log::info!("{}", msg_comp.to_pretty_console());
```

Comment:
Let put this then into world as well

Reply:
Resolved in b5e01db 🙂 
Do we plan on mimicking vanilla behavior (per-server join/quit rather than per-world) in future?

Reply:
Not sure, We could make it  a config setting, but it would be annoying. For now i would leave it per world